### PR TITLE
Cotco 227

### PR DESCRIPTION
**Pull request by andreicostin**
Add timer with extend button (all JS based right now)

### CODE REVIEW

File: app/assets/javascript/custom.js [hunk 1]
```diff
@@ -97,6 +91,40 @@ var errors = {
...
+            minutesInSeconds = minutes * 60;
+            seconds = seconds < 10 ? "0" + seconds : seconds;
+            countdown.textContent = minutesInSeconds + Number(seconds);
+        }
+
+        $('.extend-button').on('click', function() {
+            extend();
+        })
+    }
+
```
Comment (udinobi) [inline app/assets/javascript/custom.js]: A lot of blank lines :)

File: app/uk/gov/hmrc/ctc/connectors/CtcConnector.scala [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 package uk.gov.hmrc.ctc.connectors
 
-import java.time.LocalDateTime
+//import java.time.Instant
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/connectors/CtcConnector.scala]: Can this import be removed ?

Comment (andreicostin) [reply app/uk/gov/hmrc/ctc/connectors/CtcConnector.scala]: yes, but atm I am still using it when I use the dummy data

File: app/uk/gov/hmrc/ctc/connectors/CtcConnector.scala [hunk 3]
```diff
@@ -33,4 +33,17 @@ class CtcConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient)(impli
 
   def getNextTransitByPoEAndState(destinationPort: String, transitState: String)(implicit headerCarrier: HeaderCarrier): Future[Transit] =
     httpClient.GET[Transit](s"$url/transits/$destinationPort/$transitState")
+//    Future.successful(Transit(
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/connectors/CtcConnector.scala]: We have to remember to remove all these commented lines.

File: app/assets/javascript/custom.js (hunk 1 above)
Comment (andreicostin) [reply app/assets/javascript/custom.js]: I think I need to use a different IDE for my frontend stuff

File: app/uk/gov/hmrc/ctc/connectors/CtcConnector.scala (hunk 3 above)
Comment (andreicostin) [reply app/uk/gov/hmrc/ctc/connectors/CtcConnector.scala]: yes! maybe after we completely finish the whole screen

**Pull request merged**